web/public/locales/en.json

```diff
@@ -148,6 +148,17 @@
     "dataIsDelayed": "Data is delayed",
     "price": "Price"
   },
+  "aria": {
+    "label": {
+      "selectLanguage": "Select language",
+      "colorBlindMode": "Color blind mode",
+      "windLayer": "Wind layer",
+      "solarLayer": "Solar layer",
```

Comment:
```suggestion
      "colorBlindMode": "Toggle color blind mode",
      "windLayer": "Toggle wind layer",
      "solarLayer": "Toggle solar layer",
```
Either I think we should add toggle to these translations (like the suggestion) or we should add a Turn on/Turn off or Show/Hide version for these like the side panel.

Let me know what you think and once we have decided I'll add the Swedish translations and merge this.

Reply:
Hi, to give some context the reason why i chose to leave it out was that Voice Over on Mac includes it natively, both the type (toggle) aswell as the state (selected). 

![Toggle - Aria label](https://github.com/electricitymaps/electricitymaps-contrib/assets/28294647/9033a397-7d48-46a2-9969-dbe5b142f0b5)

Reply:
I was not aware of that!
Let's leave it as is then!

I'll add the Swedish translations later today.